src/server/views/lists/partials/lawyers/macros.njk

```diff
@@ -13,27 +32,27 @@
     </p>
   {% endif %}
   <p class="govuk-body govuk-!-font-size-16 govuk-!-margin-bottom-1">
-    <strong>Address</strong>: 
-      {{ item.address.firstLine }} 
+    <strong>Address</strong>:
+      {{ item.address.firstLine }}
       {{ "-" if item.address.secondLine }} {{ item.address.secondLine }}
-      {{ "-" if item.address.postCode }} {{ item.address.postCode }} 
+      {{ "-" if item.address.postCode }} {{ item.address.postCode }}
       {{ "-" if item.address.city }} {{ item.address.city }}
   </p>
-  {% if item.jsonData.email %}
+  {% if item.jsonData.emailAddress %}
     <p class="govuk-body govuk-!-font-size-16 govuk-!-margin-bottom-1">
-      <strong>Email</strong>: <a class="govuk-link" href="mailto:{{ item.jsonData.email }}">{{ item.jsonData.email }}</a>
+      <strong>Email</strong>: <a class="govuk-link" href="mailto:{{ item.jsonData.emailAddress }}">{{ item.jsonData.emailAddress }}</a>
     </p>
   {% endif %}
-  {% if item.jsonData.website %}
+  {% if item.jsonData.websiteAddress %}
     <p class="govuk-body govuk-!-font-size-16 govuk-!-margin-bottom-1">
-      <strong>Website</strong>: <a class="govuk-link" href="{{ enforceHttps(item.jsonData.website) }}" rel="noopener noreferrer" target="_blank">{{ item.jsonData.website }}</a>
```

Comment:
I know there was some discussion around enforceHttps recently - should we, shouldn't we.  Have you checked with the team that you're safe to proceed with no longer enforcing https?

Reply:
Yes. So we didn't enforce https because a lot of websites will not have been set up with https so we could have potentially ended up with a lot of broken links on the search results.

The form has validation to ensure the protocol is entered with the website address so that we link to the correct place.